functions/src/db/isTokenValidInDb.ts

```diff
@@ -0,0 +1,22 @@
+import { db } from '../firebase/firebase'
+import { UserData } from '../types/UserData'
+import { JWTData, JWTToken } from '../types/JWT'
+
+export const isTokenValidInDb = async (
+    jwtData: JWTData,
+    token: JWTToken
+): Promise<boolean> => {
+    const result = await db.collection('users').doc(jwtData.uid).get()
```

Comment:
pls consider being using a more specific variable name: isn't `result` a `userDocRef` ?

Reply:
I've changed it through it duplicate the data as the type is already a DocumentSnapshot.